common/src/main/java/org/opensearch/ml/common/parameter/Model.java

```diff
@@ -12,5 +12,5 @@
 public class Model {
     String name;
     int version;
-    byte[] content;
+    public byte[] content;
```

Comment:
Why make `content` public?

Reply:
Some unit tests in the plugin package directly use mode.content and intelliji auto detects them and set it to public. Have updated the UTs and removed the public.